stats/Export.md

```diff
@@ -28,6 +28,9 @@ parameter from the `View` and the aggregated data associated with those `columns
 
 The library SHOULD provide a means of retrieving the ViewData for any registered view in the system.
 
+ViewData is a language-specific data structure implementing the [Metrics data
```

Comment:
I think our ultimate goal is to get rid of `ViewData` and replace it with `Metric`?

/cc @bogdandrutu

Reply:
ViewData will be an intermediate step (but internal only). So from the public API will be removed but I think the implementation still needs this for accumulating data.

Reply:
How should this be re-worded? Just get rid of the mention of ViewData?

Reply:
I don't think we should mention ViewData in the spec, since it isn't visible or required.

Reply:
This file is Export.md, which existed before my PR. Not sure what the best way forward is - describe that ViewData currently exists but that it should be changed to something else in the future? What is the plan for OpenCensus Java exporters: essentially the same thing as ViewData but with a different name?

Reply:
My 2 cents:

For the first question, I think most parts of Export.md need to be re-written. From exporters' perspective, there will be no `Aggregation`, `AggregationData`, `Measure`, `View` or `ViewData`. All they can see is the `MetricSet` defined in [metric proto](https://github.com/census-instrumentation/opencensus-proto/blob/master/opencensus/proto/stats/metrics/metrics.proto#L32).

For the second question, I would say yes, the Java exporters will pull `MetricSet` instead of `ViewData` from the library.

Reply:
> This file is Export.md, which existed before my PR. Not sure what the best way forward is - describe that ViewData currently exists but that it should be changed to something else in the future? What is the plan for OpenCensus Java exporters: essentially the same thing as ViewData but with a different name?

Sorry, I didn't realize this was in the existing file.  I think that the note that you added is fine for now, and we can make more updates for the metrics API later.